### DISCUSSION TITLE
Cooldown window + BL

### DISCUSSION THREAD

**Discussion by RazaronGit** [Hekili/hekili#675] [Q&A] [answered]
Two things,

I can't seem to get the cooldown window to work with Enhancement, I'd like it to show Feral Spirits, Ascendance, etc in there but it says the window isn't currently active even though I've ticked the box in the toggle. I've also ticked the box to say show them seperately. 

Second thing is, is there any way to set up bloodlust to be apart of the rotation just for when you're in pvp or warmode on? It's just for my pvp antics when I take Shamanism, it drops the cooldown to a minute.

Thanks! 👍

**Comment by Hekili** [comment, marked as answer]
For the first part, abilities will only show in the Cooldowns display if:
1. Show Cooldowns is checked.
2. Show Separately is checked.
3. The ability is on the Cooldowns toggle.
4. The conditions for recommending the ability are met.

If all those conditions are met, and the ability isn't shown in the Cooldowns *priority display*, you can submit a ticket in the Issues section here.  Follow the instructions to provide a snapshot, and make sure the snapshot is for the Cooldowns display.

For the second part, you can look at the Wiki / Frequently Asked Questions re: making a custom priority.  You could add an entry for Bloodlust to the priority, and the Conditions would be `pvptalent.shamanism.enabled` -- this check will make sure you're in a setting where PvP talents are enabled.

**Reply by RazaronGit** [reply]
Thanks, will go ahead and give this a try. Keep up the great work!

**Reply by RazaronGit** [reply]
Updated the addon and it seems to be working now, might of been resolved. Thanks for the help! :)
